can_rgx.tex

```diff
@@ -0,0 +1,248 @@
+    \item Any unnecessary test cells in test sites will be removed by pulling on their securing Velcro.  Remove the new test cells from their holding boxes and attach them to the test sites using the Velcro.  The cell boxes will be resecured with their Velcro straps. 
+    \item Required magnets will be removed from their holding cases in the toolbox and placed into the specified test holder locations. Any magnets previously installed that are not necessary will be removed.  Needle-nosed pliers (found in toolbox) may be needed to remove magnets. 
+    \item All tools used will be returned to their designated storage place, all loose parts will be secured and the lid of the Pelican Case will be closed. 
+\end{checklist}
+\textbf{The following steps need only be performed on the first setup period and will be left on for the remainder of the flight.}
+\begin{checklist}
+    \item Once lid is closed, camera will be turned on by remote.
+    \item Interior lights will be turned on to 80\% brightness by remote. 
+\end{checklist}
+% beginning of level flight setup -- }}}
+
+% beginning of experiment parabola table -- {{{
+\definecolor{yellow3}{RGB}{255,213,79}
+\definecolor{tablegrey}{RGB}{230,230,230}
+{ \Large
+\begin{center}
+    \begin{tabular} {|l|l|c|}
+        \rowcolor{yellow3} \hline
+            Parabola \# & Test cells in use & Magnet Positions \\
+        \hline
+            1 & 1,2,3,4 & O    \\
+        \rowcolor{tablegrey} \hline
+            2 & 1,2,3,4 & L    \\
+        \hline
+            3 & 1,2,3,4 & L, R \\
+        \rowcolor{tablegrey} \hline
+            4 & 5,6,7,8 & O    \\
+        \hline
+            5 & 5,6,7,8 & L    \\
+        \rowcolor{tablegrey} \hline
+            6 & 5,6,7,8 & L, R \\
+        \hline
+            7 & 9,10,11,12 & O    \\
+        \rowcolor{tablegrey} \hline
+            8 & 9,10,11,12 & L    \\
+        \hline
+            9 & 9,10,11,12 & L, R \\
+        \rowcolor{tablegrey} \hline
+            10& 1,2,3,4 & O \\
+        \hline
+    \end{tabular}
+\end{center}
+}
+% end of experiment parabola table -- }}}
+
+% beginning of emergency operations -- {{{
+\newpage
+\subsection{Reduced Gravity and Emergency Operations}
+The following will be performed for every reduced gravity section of flight.
+Exact details about the current configuration can be found in the table above.
+
+\begin{checklist}
+
+	\item The experiment will be left to run on its own, as it is fully passive. During operation, Mission Specialists are to be watching for anything that goes wrong or needs attention. This section is intended for use in case of mishaps. 
+	\item Ferrofluid Spill/leak:
+    \begin{checklist}
+		\item The leaking cell will be identified, removed from the test site and wrapped in absorbent cloth.
+		\item Cell and cloth will be placed in a large Ziploc bag, the bag will be sealed and placed in flight toolbox. 
```

Comment:
request to say "in **the** flight toolbox"

Reply:
fixed